static/js/hotkey.js

```diff
@@ -282,6 +283,12 @@ exports.process_enter_key = function (e) {
         return true;
     }
 
+    if ($(".popover").length && $(e.target)) {
+        e.target.click();
+        status.submit_new_status();
+        return true;
+    }
+
```

Comment:
This change is not correct, according to this, `submit_new_status` is called whenever you press enter on any other popover too. The `if` condition should be such that the code gets executed only when the input box of status is focussed.

Reply:
changed to ".popover-inner .update_status_text" . Is it okay